src/seedu/addressbook/data/tag/Tag.java

```diff
@@ -50,4 +50,12 @@ public String toString() {
         return '[' + tagName + ']';
     }
 
+    /**
+     *
+     * @return tagName without the brackets
+     */
```

Comment:
The start of the header comment does not follow the convention (i.e. first sentence should be something like `Selects next line …` instead of `Select next line` or `This method is for selecting …`)

In this case perhaps just `Returns tagName without the brackets`?